docs/website/binding_types_reference_guide.md

```diff
@@ -380,6 +380,50 @@ that contains the `MakeBackgroundRed` extension method.   This means
 that you can now call "MakeBackgroundRed" on any `UIView` subclass,
 giving you the same functionality you would get on Objective-C.
 
+In some cases you will find **static** members inside categories like in the following example:
+
+```objc
+@interface FooObject (MyFooObjectExtension)
++ (BOOL)boolMethod:(NSRange *)range;
+@end
+```
+
+This will lead to an **incorrect** Category C# interface definition:
+
+```csharp
+[Category]
+[BaseType (typeof (FooObject))]
+interface FooObject_Extensions {
+
+	// Incorrect Interface definition
+	[Static]
+	[Export ("boolMethod:")]
+	bool BoolMethod (NSRange range);
+}
+```
+
+This is incorrect because in order to use the `BoolMethod` extension you need an instance of `FooObject` but you are binding an ObjC **static** extension, this is a side effect due to the fact of how C# extension methods are implemented. 
+
+The only way to use the above definitions is by the following ugly code:
+
+```csharp
+(null as FooObject).BoolMethod (range);
+```
+
+The solution to this problem is simple, just inline the `BoolMethod` definition inside the `FooObject` interface definition itself, this will allow you to call this extenion like it is intended `FooObject.BoolMethod (range)`.
+
+```csharp
+[BaseType (typeof (NSObject))]
+interface FooObject {
+
+	[Static]
+	[Export ("boolMethod:")]
+	bool BoolMethod (NSRange range);
+}
+```
+
+We will issue a warning (BI1117) whenever we find a `[Static]` member inside a `[Category]` definition. If you really want to have `[Static]` members inside your `[Category]` definitions you can silence the warning by using `[Category (ignoreStaticWarnings: true)]`.
+
```

Comment:
I admit I had not though of a parameter to avoid the warning...

However the goal of the warning is to avoid **bad** API and there are other ways to avoid it, e.g. if `[Internal]` is used then we should not issue a warning, i.e. it means this was dealt in a different way,